woocommerce/woocommerce
# Product Block Editor: use `<AttributesComboboxControl />` in the Attributes modal

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

~# Depends on https://github.com/woocommerce/woocommerce/pull/47058~

Closes https://github.com/woocommerce/woocommerce/issues/46712

This PR replaces the custom implementation for the attributes select with a new `<AttributesComboboxControl />` which isn't more than a simple wrapper of the core [<ComboboxControl />](https://wordpress.github.io/gutenberg/?path=/docs/components-comboboxcontrol--docs) component 

This PR replaces the custom implementation for the attributes select component with a new <AttributesComboboxControl /> component, which isn't more than a simple wrapper of the core  [<ComboboxControl />](https://wordpress.github.io/gutenberg/?path=/docs/components-comboboxcontrol--docs) component.






Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new Product Editor
2. Create a new product
3. Go to the Attributes section
4. Confirm it works as expected, in the same way as before
5. Go to the Variations section
6. Same. Confirm it works as expected, in the same way like before.


https://github.com/woocommerce/woocommerce/assets/77539/883a86e5-12aa-42e4-bb54-ebaf1a7ab4f7






### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
